modules/javafx.graphics/src/main/java/com/sun/javafx/application/LauncherImpl.java

```diff
@@ -658,7 +658,7 @@ private static void startToolkit() throws InterruptedException {
         final CountDownLatch startupLatch = new CountDownLatch(1);
 
         // Note, this method is called on the FX Application Thread
-        PlatformImpl.startup(() -> startupLatch.countDown(), true);
+        PlatformImpl.startup(() -> startupLatch.countDown());
```

Comment:
Glad to see this reverted. I was going to ask you why it was needed (it shouldn't be).

Reply:
It was added, because I thought it shouldn't be allowed to call Application.launch after Platform.startup.